### PR TITLE
Mesos 0.23.0

### PR DESCRIPTION
**Pull request by ssalinas**
@tpetr @wsorenson

### CODE REVIEW

**Comment by tpetr** [comment]
when you have a chance, can you rebase? those first 3 commits shouldn't be in the PR

File: pom.xml [hunk 1]
```diff
@@ -35,8 +35,10 @@
     <basepom.release.profiles>oss-release,build-swagger-documentation</basepom.release.profiles>
 
     <baragon.version>0.1.4</baragon.version>
-    <mesos.docker.tag>0.21.1-1.1.ubuntu1404</mesos.docker.tag> <!-- 0.21.0-1.0.ubuntu1404 is missing for mesos-slave -->
+    <mesos.docker.tag>0.23.0-1.0.ubuntu1404</mesos.docker.tag> <!-- 0.21.0-1.0.ubuntu1404 is missing for mesos-slave -->
```
Comment (tpetr) [inline pom.xml]: comment unnecessary here

**Comment by ssalinas** [comment]
@tpetr merged master in and removed that comment, diff is now just the appropriate commits

File: SingularityBase/src/main/java/com/hubspot/mesos/SingularityDockerInfo.java [hunk 2]
```diff
@@ -14,29 +13,22 @@
...
+  private final boolean forcePullImage;
+  private final Map<String, String> parameters;
 
   @JsonCreator
   public SingularityDockerInfo(@JsonProperty("image") String image,
                                @JsonProperty("privileged") boolean privileged,
                                @JsonProperty("network") SingularityDockerNetworkType network,
-                               @JsonProperty("portMappings") Optional<List<SingularityDockerPortMapping>> portMappings) {
+                               @JsonProperty("portMappings") Optional<List<SingularityDockerPortMapping>> portMappings,
+                               @JsonProperty("forcePullImage") boolean forcePullImage,
```
Comment (tpetr) [inline SingularityBase/src/main/java/com/hubspot/mesos/SingularityDockerInfo.java]: consider `Optional<Boolean>` for backwards compatibility

**Comment by ssalinas** [comment]
@tpetr made that boolean optional, and also added a constructor that matched the previous version, marked as deprecated

**Pull request merged**